src/components/DecisionDetail/VoteActions.js

```diff
@@ -40,22 +40,24 @@ const VoteActions = React.memo(({ vote, onVoteYes, onVoteNo, onExecute }) => {
     startTimestamp,
   } = useExtendedVoteData(vote)
 
-  console.log('AFTER DATAAAAAAA', canUserVote, canExecute)
-
   const hasVoted = [VOTE_YEA, VOTE_NAY].includes(connectedAccountVote)
 
   useEffect(() => {
     let cancelled = false
 
     const whenReady = async () => {
-      await Promise.all([
-        canUserVotePromise,
-        canExecutePromise,
-        userBalancePromise,
-        userBalanceNowPromise,
-      ])
-      if (!cancelled) {
-        setReady(true)
+      try {
```

Comment:
ah, was this failing?  did you get to see the reason?

Reply:
Was having an error that could fix afterwards, but i also thought it would be a good idea to catch the errors